phantom.js

```diff
@@ -1,7 +1,7 @@
-// Generated by CoffeeScript 1.9.2
```

Comment:
Why did you down grade coffee script? Can you use at least the same version please?